tests/console/docker.pm

```diff
@@ -28,8 +28,13 @@ use strict;
 sub run {
     select_console("root-console");
 
-    # install the docker package
-    zypper_call("in docker");
+    if (check_var("DISTRI", "caasp")) {
```

Comment:
-1 - You need to test this on Kubic as well as CaaSP :)

Please either check for caasp || kubic or create a "is_casp" sub in the main.pm so you can use it here and we can reuse it when this problem comes up

Reply:
FYI https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/utils.pm#L264